### ISSUE TITLE
[Feature Request] v-loading component support does not support global configuration

### ISSUE THREAD

**Issue by xuliangzhan** [ElemeFE/element#13599]
### Existing Component
是

### Component Name
Loading

### Description
 项目中有n个页面需要用到loading，如果每个页面都这么写，后续如果涉及改动文字或样式，又得全局替换比较麻烦；希望v-loading能支持全局默认配置。

 <div
    v-loading="loading"
    element-loading-text="加载中..."
    element-loading-spinner="el-icon-loading"
    element-loading-background="rgba(0, 0, 0, 0.8)">
  ...
</div>

**Comment by wacky6** [comment]
http://element.eleme.io/#/zh-CN/component/loading#fu-wu

**Comment by Zver64** [comment]
I've checked the documentation, but didn't find any solution of how to change settings for all loading instances in one place.
Let's say I want to change default background to black color. So if I create a new loading instance with empty settings, then the background color should be black. Or if I use v-loading directive without "Element-loading-background", then also the background color should be black.
I've tried to pass this settings when using Vue.use(Loading) and
  Vue.use(Loading.directive) like this:
```
Vue.use(Loading, {background: 'rgba(0,0,0,0.9)'});
  Vue.use(Loading.directive, {background: 'rgba(0,0,0,0.9)'});

```
But it doesn't work.
